app/models/user_student.rb

```diff
@@ -12,6 +12,9 @@ def welcome_banner_mark_as_viewed!
     user.update!(welcome_banner_viewed: true)
   end
 
+  delegate :degree_subjects, to: :user
+  delegate :degree_subject_groups, to: :user
```

Comment:
nit: aca tmb podes unirlos
```suggestion
  delegate :degree_subject_groups, :degree_subjects, to: :user
```